### DISCUSSION TITLE
Hand off calls between devices

### DISCUSSION THREAD

**Discussion by HarHarLinks** [element-hq/element-meta#229] [Ideas]
I want to be on a native (at this point e2ee DM) call on element desktop on PC and then get up and continue on my phone seamlessly, without dropping/restarting the call.

**Comment by kittykat** [comment]
Yes, I'd really like this to be a feature! It would be very handy when I need to step away from my desk (e.g. to get the door for a package delivery)

**Comment by robintown** [comment]
We'll get this for free once we address https://github.com/vector-im/element-call/issues/47 and then start using Element Call for native calls on Web
